openedx/core/djangoapps/heartbeat/urls.py

```diff
@@ -0,0 +1,10 @@
+"""
```

Comment:
Any idea why this isn't showing up as a move? It isn't a big deal since the code is so simple...